### PR TITLE
dir: update Dutch translation

### PR DESCRIPTION
**Pull request by sebastiaanspeck**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.nl/windows/dir.md [hunk 1]
```diff
@@ -1,20 +1,24 @@
...
 
 - Geef de inhoud weer van de huidige map:
 
 `dir`
 
-- Geef de inhoud weer van een gegeven directory:
+- Geef de inhoud weer van een gegeven map:
 
-`dir {{pad/naar/directory}}`
+`dir {{pad\naar\map}}`
```
Comment (blueskyson) [inline pages.nl/windows/dir.md:12]: ref: [common-argument.md](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/translation-templates/common-arguments.md)
```suggestion
`dir {{pad\naar\directory}}`
```

File: pages.nl/windows/dir.md [hunk 2]
```diff
@@ -1,20 +1,24 @@
...
+- Geef de inhoud weer van de huidige map, inclusief verborgen bestanden:
 
-`dir /A`
+`dir /a`
 
-- Geef de inhoud weer van een gegeven directory, inclusief verborgen bestanden:
+- Geef de inhoud weer van een gegeven map, inclusief verborgen bestanden:
 
-`dir {{pad/naar/directory}} /A`
+`dir {{pad\naar\map}} /A`
```
Comment (blueskyson) [inline pages.nl/windows/dir.md:20]:
```suggestion
`dir {{pad\naar\directory}} /A`
```

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Review by blueskyson — APPROVED** [review]

**Review by kbdharun — COMMENTED** [review]
Now, that the PR got merged. Can you revert `directory` back to the native name (i.e `map`)?

**Review by kbdharun — COMMENTED** [review]
Now, that the PR got merged. Can you revert `directory` back to the native name (i.e `map`)?

**Comment by sebastiaanspeck** [comment]
Will be picked up in https://github.com/tldr-pages/tldr/pull/11030

**Pull request merged**